api.md

```diff
@@ -1229,6 +1230,39 @@ For a table with more than one space dimension:
 SELECT set_number_partitions('conditions', 2, 'device_id');
 ```
 
+
+---
+
+## set_replication_factor() [](set_replication_factor)
+Sets the replication factor of a distributed hypertable to the given value. 
+The setting will affect only new chunks. 
+Newly created chunks of the hypertable will be replicated to different data nodes,
+so the amount of replicas is the same as the replication factor.
+
+If existing chunks have less replicas than the given replication factor,
+the function will print a warning.
+
+#### Required Arguments [](set_replication_factor-required-arguments)
+
+|Name|Description|
+|---|---|
+| `main_table` | Identifier of the distributed hypertable to update the replication factor for.|
```

Comment:
Why "main" table? How does it differ from a non-"main" table? I think you should just say "hypertable". I do not think you need to talk about the "identifier". Suggestion:

> Hypertable to change replication factor for.

Reply:
> Why "main" table? How does it differ from a non-"main" table? I think you should just say "hypertable". I do not think you need to talk about the "identifier". Suggestion:
> 
> > Hypertable to change replication factor for.

This is the actual name of the argument in the code. I agree that it is not meaningful enough name, however it is not an uncommon case (see [set_number_partitions](https://docs.timescale.com/clustering/api#set_number_partitions), which I used for inspiration). IMHO this name should consistent with the name in the code and it is not worth to change the code just for the argument name.

Reply:
Legacy name is a bit unfortunate, but I believe it comes from `create_hypertable()` where naming the parameter `hypertable` doesn't make sense since it is not yet a hypertable at that point. Would probably be better to just name it `table` in all functions but should probably keep legacy name for consistency.

Fortunately, users will probably never reference this parameter explicitly since it is the first one.

Reply:
If the implementation will be changed, it will be good opportunity to address this comment. It will be different PR the most likely.